packages/DynamicHyperlink/src/DynamicHyperlink.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import PropTypes from "prop-types";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import "./DynamicHyperlink.scss";
+
+export default function DynamicHyperlink({ onFetch }) {
+  const I18n = useI18n();
+
+  React.useEffect(() => {
+    function updateDynamicHyperlinks() {
+      const dynamicHyperlinks = Array.from(document.querySelectorAll("[data-dynamic-hyperlink]"));
+
+      dynamicHyperlinks
```

Comment:
will this be able to handle multiple dynamic links in parallel?  lets say that there are 50 links on the page...will they all get spinners and wait for API response to come in?

Reply:
yes